File: FrontEnd/src/app/models/respuestaCuestionario.ts
import { RespuestaCuestionarioDetalle } from "./respuestaCuestionarioDetalle";

export class RespuestaCuestionario{
    cuestionarioId!:number;
    nombreParticipante!:string;
    listRtaCuestionarioDetalle!:RespuestaCuestionarioDetalle[];
}

File: Back End/Demo.Entities/RespuestaCuestionario.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Demo.Entities
{
    public class RespuestaCuestionario:EntityBase
    {
        [Required]
        [Column(TypeName ="varchar(100)")]
        public string NombreParticipante { get; set; }
        public DateTime Fecha { get; set; }
        public int Activo { get; set; }
        public int CuestionarioId { get; set; }
        public Cuestionario? Cuestionario { get; set; }
        public List<RespuestaCuestionarioDetalle> ListRtaCuestionarioDetalle { get; set; }
    }
}
